mmdet/core/bbox/coder/delta_xywh_bbox_coder.py

```diff
@@ -70,6 +71,8 @@ def decode(self,
         """
 
         assert pred_bboxes.size(0) == bboxes.size(0)
+        if pred_bboxes.ndim == 3:
+            assert pred_bboxes.size(1) == bboxes.size(1)
```

Comment:
Should also test size(0)?

Reply:
The shape may be (B, N, 4) or (N, 4), so I think it might be asserted.